PROCESS.md

```diff
@@ -0,0 +1,279 @@
+# Haskell Security Response Team Process
+
+> Copyright 2017, OpenStack Vulnerability Management Team
+> Copyright 2023, Haskell Security Response Team
+> This work is licensed under a Creative Commons Attribution 3.0 Unported License.
+>   http://creativecommons.org/licenses/by/3.0/legalcode
+> Original version: https://opendev.org/openstack/ossa/src/branch/master/doc/source/vmt-process.rst
+
+The Haskell Security Response Team (HSRT) is responsible for coordinating
+the progressive disclosure of a vulnerability.
+
+Members of the team are independent and security-minded folks who ensure
+that vulnerabilities are dealt with in a timely manner and that
+downstream stakeholders are notified in a coordinated and fair manner.
+Where a member of the team is employed by a downstream stakeholder, the
+member does not give their employer prior notice of any vulnerabilities.
+In order to reduce the disclosure of vulnerability in the early stages,
+membership of this team is intentionally limited to a small number of
+people.
+
+## Supported versions
+
+The Vulnerability Management team coordinates patches fixing
```

Comment:
I actually think that we should say that the role of the team is to coordinate responsible disclosure of security vulnerabilities in the Haskell ecosystem and help Haskell users have an accurate and timely assessment of risks.